### PR TITLE
CB-11244 : cordova-common tests for ConfigFile

### PR DESCRIPTION
**Pull request by audreyso**

### CODE REVIEW

**Comment by infil00p** [comment]
So, this has #7 in it, right? Does it make sense to just merge this in and close #7?

**Comment by audreyso** [comment]
@infil00p Yes I wanted to make sure the tests were passing so I pulled down your changes. If it's okay with you we can use this PR (#8)? Wasn't sure the best way to do it!

File: src/ConfigChanges/ConfigFile.js [hunk 1]
```diff
@@ -184,29 +184,36 @@ function resolveConfigFilePath (project_dir, platform, file) {
...
+    // be resolved to the real location of the file.
+    // TODO: Move this out of cordova-common into platforms
     if (file === 'config.xml') {
         if (platform === 'ubuntu') {
             filepath = path.join(project_dir, 'config.xml');
         } else if (platform === 'ios') {
-            var iospath = getIOSProjectname(project_dir);
+            var iospath = module.exports.getIOSProjectname(project_dir);
             filepath = path.join(project_dir, iospath, 'config.xml');
-        } else if (platform === 'android') {
```
Comment (stevengill) [inline src/ConfigChanges/ConfigFile.js]: I chatted with @infil00p yesterday and he is going to re-add lines 195 and 196. Then this is good to merge

Comment (infil00p) [reply src/ConfigChanges/ConfigFile.js]: Wait, I think I misunderstood this.  Why would this fail?

Comment (infil00p) [reply src/ConfigChanges/ConfigFile.js]: Actually, this code needs to be fixed up.  Sent a new PR to #7 to check against.

**Comment by stevengill** [comment]
@audreyso Can you rebase this off master now that #7 has been merged?

**Comment by audreyso** [comment]
yes done!

**Pull request closed without merging**